recipes/h5py/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.6.0" %}
+
+package:
+  name: h5py
+  version: {{ version }}
+
+source:
+  fn: h5py-{{ version }}.tar.gz
+  url: https://github.com/h5py/h5py/archive/{{ version }}.tar.gz
+  md5: 613c5776192eaf02ff96a82dfc6d2e78
+
+requirements:
+  build:
+    - python
+    - numpy
+    - hdf5
+    - cython
+    - six
+    - pkgconfig
+
+  run:
+    - python
+    - numpy
+    - hdf5
+    - six
+    - unittest2    #[py26]
```

Comment:
I am fine with that. Unused in conda-forge, but leaves it general to other users in other places.